spec/index.bs

```diff
@@ -3458,29 +3458,26 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
                                 [=exceeds the binding slot limits|exceed the binding slot limits=]
                                 of |this|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.
                             - For each {{GPUBindGroupLayoutEntry}} |entry| in |descriptor|.{{GPUBindGroupLayoutDescriptor/entries}}:
-                                - Let |bufferLayout| be |entry|.{{GPUBindGroupLayoutEntry/buffer}}
-                                - Let |samplerLayout| be |entry|.{{GPUBindGroupLayoutEntry/sampler}}
-                                - Let |textureLayout| be |entry|.{{GPUBindGroupLayoutEntry/texture}}
-                                - Let |storageTextureLayout| be |entry|.{{GPUBindGroupLayoutEntry/storageTexture}}
-
-                                - Exactly one of |bufferLayout|, |samplerLayout|, |textureLayout|,
-                                    or |storageTextureLayout| are not `undefined`.
-
+                                - Exactly one of |entry|["buffer"], |entry|["sampler"], |entry|["texture"], or
+                                    |entry|["storageTexture"] [=map/exists=].
```

Comment:
We use `.buffer` instead of `["buffer"]` elsewhere so I think we can do so here too for consistency.

However of course, the upstream spec only defines what `["buffer"]` means, we define the `.` ("dot syntax") ourselves in this spec. Arguably we should define what the dot syntax means on [=map=]s.